cmd/describe/autoscaler/cmd.go

```diff
@@ -0,0 +1,66 @@
+package autoscaler
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/rosa/pkg/clusterautoscaler"
+	"github.com/openshift/rosa/pkg/ocm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+)
+
+const (
+	use     = "autoscaler"
+	short   = "Show details of the autoscaler for a cluster"
+	long    = short
+	example = ` # Describe the autoscaler for cluster 'foo'
+rosa describe autoscaler --cluster foo`
+)
+
+func NewDescribeAutoscalerCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     use,
+		Short:   short,
+		Long:    long,
+		Example: example,
+		Args:    cobra.NoArgs,
+		Run:     rosa.DefaultRunner(rosa.DefaultRuntime(), DescribeAutoscalerRunner()),
+	}
+
+	output.AddFlag(cmd)
+	ocm.AddClusterFlag(cmd)
+	return cmd
+}
+
+func DescribeAutoscalerRunner() rosa.CommandRunner {
+	return func(_ context.Context, runtime *rosa.Runtime, _ *cobra.Command, _ []string) error {
+		cluster, err := runtime.OCMClient.GetCluster(runtime.ClusterKey, runtime.Creator)
+		if err != nil {
+			return err
+		}
+
+		err = clusterautoscaler.IsAutoscalerSupported(runtime, cluster)
+		if err != nil {
+			return err
+		}
+
+		autoscaler, err := runtime.OCMClient.GetClusterAutoscaler(cluster.ID())
+		if err != nil {
+			return err
+		}
+
+		if autoscaler == nil {
+			return fmt.Errorf("No autoscaler exists for cluster '%s'", runtime.ClusterKey)
+		}
```

Comment:
Does it make sense to decompose this further into a new `ClusterAutoscalerService` so that the command becomes even briefer e.g:

```
service := NewClusterAutoscalerService(runtime.OCMClient)
supported, err = service.IsAutoscalerSupported(runtime, cluster)

autoscaler, err := service.GetClusterAutoscaler(cluster.ID())
if err != nil {
return err
}
```

Reply:
I think it's ok to keep it that way, it seems like we return nil for `StatusNotFound`, and have different error messages per command.
I wish all the commands in ROSA were as brief as this one:)

With that being said, we could return a boolean from the function for `exist` and check the boolean value instead of nil to improve readability (maybe outside of the scope of this PR).